packages/@coorpacademy-player-services/src/types.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+import type {Action, Content} from '@coorpacademy/progression-engine';
+
+type Url = string;
+
+type JwpOptions = {
+  customProps: {
+    aspectratio: '16:9' | '4:3',
+    autostart: boolean,
+    skin: {
+      name: string
+    },
+    width: string
+  },
+  file: Url,
```

Comment:
Is a file an URL ? or a path to a file ?

Reply:
url
```
 "file": "https://simoocdigital.credit-agricole.fr/media/content/bigdata/159363386.mp4",
```